packages/fast-form-generator-react/src/form/form-item.align-horizontal.spec.tsx

```diff
@@ -60,6 +60,17 @@ describe("AlignHorizontal", () => {
         expect(handleChange).toHaveBeenCalled();
         expect(handleChange.mock.calls[0][1]).toEqual("left");
     });
+    test("should be disabled if disabled props is passed", () => {
+        const rendered: any = mount(
+            <AlignHorizontal {...alignHorizontalProps} disabled={true} />
+        );
+        const inputs: any = rendered.find("input");
```

Comment:
I think we have a pattern of rendered being "any", but here it seems like you are finding HTMLInputElements. Is there a typing we can leverage here?

Reply:
ShallowWrapper is returned from find(). I will add to dev dependencies @types/enzyme which can help with the typing.